src/mv_oc_chef_authz.erl

```diff
@@ -496,7 +496,9 @@ set_ace_for_entity(RequestorId, AuthzType, Id, AccessMethod,
     case mv_oc_chef_authz_http:request(Url, put, [], Body, EffectiveRequestorId) of
         ok -> ok;
         %% Expected errors are forbidden, not_found, server_error
-        {error, Error} -> {error, Error}
+        {error, Error} ->
+            lager:error("Failed to set ACE. Error: ~p  Url: ~p  Body: ~p", [Error, Url, Body]),
+            {error, Error}
```

Comment:
This is cool, we should make the same change in https://github.com/chef/chef-server/blob/master/src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_org_creator.erl some time